monkey/monkey_island/cc/server_utils/consts.py

```diff
@@ -14,7 +15,7 @@ def get_default_data_dir() -> str:
 
 SERVER_CONFIG_FILENAME = "server_config.json"
 
-MONKEY_ISLAND_ABS_PATH = os.path.join(os.getcwd(), "monkey_island")
+MONKEY_ISLAND_ABS_PATH = str(Path(__file__).resolve().parent.parent.parent)
```

Comment:
This now relies on directory structure. Is there a way to add a unit test that would fail if we move `consts.py` up/down a directory?

Reply:
I could `assert MONKEY_ISLAND_ABS_PATH.endswith("monkey_island")`